pkgs/servers/scylladb/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv,
+  fetchgit,
+  python3Packages,
+  python3,
+  pkgconfig,
+  gcc8Stdenv,
+  boost,
+  git,
+  systemd,
+  gnutls,
+  cmake,
+  makeWrapper,
+  ninja,
+  ragel,
+  hwloc,
+  jsoncpp,
+  libantlr3cpp,
+  antlr3,
+  numactl,
+  protobuf,
+  cryptopp,
+  libxfs,
+  libyamlcpp,
+  libsystemtap,
+  lksctp-tools,
+  lz4,
+  libxml2,
+  zlib,
+  libpciaccess,
+  snappy,
+  libtool,
+  thrift-0_10
+}:
+gcc8Stdenv.mkDerivation rec {
+  pname = "scylladb";
+  version = "3.0.5";
+
+  src = fetchgit {
+    url = "https://github.com/scylladb/scylla.git";
+    rev = "403f66ecad6bc773712c69c4a80ebd172eb48b13";
+    sha256 = "14mg0kzpkrxvwqyiy19ndy4rsc7s5gnv2gwd3xdwm1lx1ln8ywsi";
+    fetchSubmodules = true;
+  };
+
+  patches = [ ./seastar-configure-script-paths.patch ];
+
+  nativeBuildInputs = [
+   pkgconfig
+   cmake
+   makeWrapper
+   ninja
+  ];
+
+  buildInputs = [
+   python3Packages.pyparsing
+   python3
+   boost
+   git
+   systemd
+   gnutls
+   ragel
+   hwloc
+   jsoncpp
+   libantlr3cpp
+   numactl
+   antlr3
+   protobuf
+   cryptopp
+   libxfs
+   libyamlcpp
+   libsystemtap
+   lksctp-tools
+   lz4
+   libxml2
+   zlib
+   libpciaccess
+   snappy
+   libtool
+   thrift-0_10
+  ];
+
+  configurePhase = ''
+     patchShebangs ./configure.py
+    ./configure.py --mode=release
+  '';
+  buildPhase = ''
+    ninja -j "$NIX_BUILD_CORES" -v
```

Comment:
I thought `ninja` build "just work" with `nix`. Any specific reason for this?

Reply:
You are absolutely Sir, it does work out of the box! Removed the `buildPhase` override entirely.